Makefile

```diff
@@ -5,7 +5,7 @@ env:
 	conda create -n $(BASENAME)  python=$(PYTHON)
 
 setup:
-	conda install --file requirements.txt
+	conda install --file requirements.txt -c conda-forge
```

Comment:
How about adding a variable "CONDA_CH" in the top line?
Because some people want to add another channel. 

e.g. 
```
PYTHON=3.8
BASENAME=$(shell basename $(CURDIR))
CONDA_CH = conda-forge
```

Reply:
`-c` doesn't accept multiple values.
Please let me know if there is another better way.

Reply:
or we can consider this when we should consider multiple channels.

Reply:
How about use "addprefix" function. 
```
PYTHON=3.8
BASENAME=$(shell basename $(CURDIR))
CONDA_CH = conda-forge pytorch

env:
        conda create -n $(BASENAME)  python=$(PYTHON)

setup:
        conda install --file requirements.txt $(addprefix -c ,$(CONDA_CH))
        pre-commit install
```

I referred to this documentation.
https://www.gnu.org/software/make/manual/html_node/File-Name-Functions.html

Reply:
Good idea. I modified.